### ISSUE TITLE
Updating helm autocomplete menu for Layouts

### ISSUE THREAD

**Issue by smimish** [syl20bnr/spacemacs#7597]
I think helm autocomplete SPC l for the "L" should go into the "Actions" section and be renamed to "load layouts from file". This will make it consistent with "s/S" actions, which save it to the file.

This will fix 2 issues with it:
- The current wording is not very clear, it takes time to find it.
- The location should be under the Actions, where you would naturally look for it.

**Comment by bmag** [comment]
That's not helm autocomplete, that's the layouts transient state. I think your suggestion makes sense, PR welcome.

**Comment by pawandubey** [comment]
@bmag can you help me out a little bit with what exactly has to be modified here as it is not clear from the OP.

I see that in `spacemacs-layouts/packages.el`, the leader key for is set  :

````
(spacemacs/set-leader-keys "l" 'spacemacs/layouts-transient-state/body)
````

What does this need to be changed to?

**Comment by TheBB** [comment]
It doesn't need to be changed. The transient state is the stuff before that. Specifically it sounds like you want to change the doc hint [here](https://github.com/syl20bnr/spacemacs/blob/develop/layers/+spacemacs/spacemacs-layouts/packages.el#L131-L147)

**Comment by smimish** [comment]
@pawandubey this is not about changing the leader key by itself or its action, but rather changing its description text in the helm text. 

See the picture:

![image](https://cloud.githubusercontent.com/assets/4650168/19862984/62c2231e-9f8a-11e6-8ba2-01280f54cbdf.png)

**Comment by pawandubey** [comment]
Got it. Will do.

**Comment by pawandubey** [comment]
So I made a PR. But the output as seen in the buffer is a bit wonky.
![](http://i.imgur.com/Ik3lLtu.png)

**Comment by smimish** [comment]
One option would be to merge a/A actions with this description:
add buffer / add all buffers
Those are similar on a way s/S are similar

**Comment by pawandubey** [comment]
I think the separate a/A options are fine. What I don't understand are why the last two options in the Actions column are displaced.

**Comment by smimish** [comment]
Maybe your editor is not set for monospace font, or there are tabs somewhere. Show gist...

**Comment by TheBB** [comment]
I explained in the PR.

**Comment by Miciah** [comment]
This was fixed with #7609 / https://github.com/syl20bnr/spacemacs/commit/83553a54dacfcd8721e8934ec9d9c0e24407804b.  Can the issue be labeled as "Fixed in develop"?
